xarm_moveit_config/config/xarm6/joint_limits.yaml

```diff
@@ -2,32 +2,32 @@
 # Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
 # Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
 joint_limits:
-  joint1:
+  xarm6_joint1:
```

Comment:
This has to match URDF joint names (after xacro substitution). Move to description or similar package in the future.